src/renderer/components/Delegation/ValidatorRow.js

```diff
@@ -221,6 +227,37 @@ const ValidatorRow = ({
 
   const itemExists = typeof value === "number";
 
+  const InputComponent =
```

Comment:
this is not a component. it's an element. should be lower cased. can be called `const input =`